src/action/send/canvasemail.py

```diff
@@ -86,7 +86,7 @@ def send_canvas_emails(
         # https://canvas.instructure.com/doc/api/conversations.html
         #
         canvas_email_payload = {
-            'recipients[]': msg_to,
+            'recipients[]': int(msg_to),
```

Comment:
This is error appears if the column has changed to "double" instead of "integer" internally. The change has been merged in development.